plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/TestPinotLatestNoGrpcConnectorSmokeTest.java

```diff
@@ -15,8 +15,8 @@
 
 import static io.trino.plugin.pinot.TestingPinotCluster.PINOT_LATEST_IMAGE_NAME;
 
-public class TestPinotWithoutAuthenticationIntegrationLatestVersionNoGrpcConnectorSmokeTest
```

Comment:
After i made this class names a little shorter i realized we don't have non-Smoke BCT coverage for Pinot yet.
@elonazoulay is there an issue for this already?

Reply:
There is no issue but I can create a BCT for pinot.